dev-haskell/network/Manifest

```diff
@@ -1,3 +1,4 @@
 DIST network-3.0.1.1.tar.gz 114777 BLAKE2B 05ced39ff17eb68e5e70ddb1036770413224839d5a2c84ee568823feb364f1da898bdc56ae7e64f0e24db80dbc2ec739e42763708b3373f49a85ecf911c72621 SHA512 faca3b25c8724a0c832b6e12e21381fa9642a60bce8a01ea7b570216cd429b2332b14903011f4a554af227de5e502f9619860c230a8f2a69cb5e2289b1947e3d
 DIST network-3.1.1.1.tar.gz 118478 BLAKE2B 17358583c7b62168570f5c315ed1cfac07988c34dd0c17c2c43365f0da46f04b1d18b2a3f4940d46a22a78b3cc6baabe1f996dae636e78edae865520812c2ce6 SHA512 caafd3c0730d0f6378230e0acdf285ca50480f02a44963c1f9ae9dc3d53cd496dd3171d7ff4bf79a1e072b85698d74f82d8075b999d5ba1d17ab1429c8094ca8
 DIST network-3.1.2.5.tar.gz 139267 BLAKE2B 682726d42db1d899810fbe6c0aa89f1b25770a46041a9136260d104f658611bc747c4695f403ae52234ac3506221619d7741e75bea987d0cc19e6a5066172f9f SHA512 671429b18f11285b157c70e669051f57a30bf5e85ada1134464f80dd978f71878f8b4d2564242c9ff4bb100b49af771e449c556654486f5eb433f39401ec6c00
+DIST network-3.1.2.6.tar.gz 141220 BLAKE2B 69ceb7df00b962ed472def6ec2a3d566bb5671ae676d6c7222dae4759f8c715ad53b6aece9a821144dc437b6f3ebbb5fd5b7c6c12f65744f8f65899415a88fca SHA512 1fe886ae98ef8cd6cfd62c3cd5570df2befd50cfb9594c16b7b786d5f87c069d1c91aea3b911a7be7387bca5cc51b5efc4176bcf77a29863aa27342932714c71
```

Comment:
Would you please:

1. Remove `network-3.1.2.5.tar.gz` and update the manifest (`pkgdev manifest`), then amend/fixup these changes to your latest commit. This will give a nice diff view between the last version and your new version in github.
2. Remove `network-3.0.1.1.tar.gz` and `network-3.1.1.1.tar.gz`, then update the manifest, and add these changes as a _new_ commit (which will clear out the rest of the old versions).

Reply:
Done.  (I don't have `pkgdev`, so I used `ebuild network-3.1.2.6.ebuild manifest` instead.)

Should I also update the patch to the one which was eventually accepted by upstreams (main difference is macOS support)?

Reply:
Sure